L1Trigger/L1THGCal/python/customTriggerSums.py

```diff
@@ -0,0 +1,32 @@
+import FWCore.ParameterSet.Config as cms
+import math
```

Comment:
Why do we need math here? 
Thx.

Reply:
Indeed it doesn't seem we need it.
I'll remove it.
Thanks!